src/libostree/ostree-repo.c

```diff
@@ -4452,12 +4452,13 @@ ostree_repo_query_object_storage_size (OstreeRepo           *self,
  * @self: Repo
  * @objtype: Object type
  * @sha256: ASCII checksum
- * @out_variant: (out) (transfer full): Metadata
+ * @out_variant: (out) (nullable) (transfer full): Metadata
  * @error: Error
  *
  * Attempt to load the metadata object @sha256 of type @objtype if it
  * exists, storing the result in @out_variant.  If it doesn't exist,
- * %NULL is returned.
+ * @out_variant will be set to %NULL and the function will still
```

Comment:
Hmm, AFAICT `*out_variant` isn't explicitly set to `NULL` in the `ENOENT` case.

Reply:
Good catch!  In practice for callers they usually need to zero-initialize anyways for reliability, but indeed let's be explicit here.